docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -10,9 +10,7 @@ to save power and free CPU resources.
 
 The GNA plugin provides a way to run inference on Intel® GNA, as well as in the software execution mode on CPU.
 
-## Devices with Intel® GNA
-
-Devices with Intel® GNA support:
+## Platforms with Intel® GNA support
```

Comment:
Can we make this list with platforms foldable? This is good reference info, but perhaps not entirely necessary for all and takes quite some space.

Reply:
@yury-gorbachev As @ilya-lavrenov proposed there is sense to move these links to table on **Working with devices** page to avoid duplication